pkg/aspectsupport/uber/env_test.go

```diff
@@ -24,5 +24,5 @@ func TestEnv(t *testing.T) {
 	log := env.Logger()
 	log.Infof("Test%s", "ing")
 	log.Warningf("Test%s", "ing")
-	log.Errorf("Test%s", "ing")
+	_ = log.Errorf("Test%s", "ing")
```

Comment:
Yes, there's a linter that demands you acknowledge that functions return errors and don't drop 'em on the floor. In general, that's a good thing IMO.